javascript-operators-and-expressions/main.js

```diff
@@ -0,0 +1,39 @@
+const width = 5;
+const height = 10;
+const area = width * height;
+console.log('Value of area: ', area);
+console.log('typeof area: ', typeof area);
+
+const bill = 11;
+const payment = 4;
+const change = bill - payment;
```

Comment:
Your `payment` should be larger than your `bill`. You'll also need to swap line 9. It should be `payment - bill`

Reply:
<img width="284" alt="image" src="https://user-images.githubusercontent.com/79122214/236952367-afb6eccb-15dd-423b-9823-837ace252fec.png">